invenio/modules/relationships/api.py

```diff
@@ -0,0 +1,6 @@
+from werkzeug.utils import import_string
+
+from .config import CFG_RELATIONSHIPS_NODE_ENGINE, CFG_RELATIONSHIPS_EDGE_ENGINE
+
+Node = import_string(CFG_RELATIONSHIPS_NODE_ENGINE)
```

Comment:
I think you should use `app.config`. In the usual case, the config file is not overwritten, there is additional config file outside of the source of `Invenio`.